offlinevalidator/offlinevalidator.go

```diff
@@ -87,11 +85,51 @@ func Validate(context *context.Context) bool {
 	resources := obtainResources(deNilizedTemplate, perunTemplate, context.Logger)
 	deadResources := getNilResources(resources)
 	deadProperties := getNilProperties(resources)
+	hasAllowedValues := findingAllowedValues(goFormationTemplate, context.Logger)
+	if hasAllowedValues == true {
+		return false
+	}
 
 	valid = validateResources(resources, &specification, context.Logger, deadProperties, deadResources)
 	return valid
 }
 
+func findingAllowedValues(goFormationTemplate cloudformation.Template, logger *logger.Logger) bool {
```

Comment:
Name this function `hasAllowedValues` - if function returns boolean it is good practise to name boolean functions similar to questions that can be answered `yes` or `no`.